lib/guest_installation_and_configuration_base.pm

```diff
@@ -2044,6 +2096,7 @@ sub do_detach_guest_installation_screen {
         record_info("Failed to detach $self->{guest_name} installation screen process $self->{guest_installation_session}", "Bad luck !");
         reset_consoles;
         select_console('root-ssh');
+        alp_workloads::kvm_workload_utils::enter_kvm_container_sh if version_utils::is_alp;
         $self->get_guest_installation_session if ($self->{guest_installation_session} eq '');
         type_string("reset\n");
         wait_still_screen;
```

Comment:
Make sure your verification on these changes in do_detach_guest_installation_screen and do_detach_guest_installation_screen are bulletproof.

Reply:
Yes, they have been well verified(and fixed) in devel tests which are more than the validation jobs listed here.